cli/outputflags/flags.go

```diff
@@ -26,7 +26,15 @@ func (f *Flags) Options() zio.WriterOpts {
 	return f.WriterOpts
 }
 
+func (f *Flags) formatFlags(fs *flag.FlagSet) {
+	fs.StringVar(&f.Format, "f", "zng", "format for output data [zng,zst,ndjson,table,text,csv,zeek,zjson,zson,tzng]")
+	fs.BoolVar(&f.textShortcut, "t", false, "use format tzng independent of -f option")
+	fs.BoolVar(&f.forceBinary, "B", false, "allow binary zng be sent to a terminal output")
+}
+
 func (f *Flags) setFlags(fs *flag.FlagSet) {
+	f.formatFlags(fs)
```

Comment:
This causes `SetFlagsWithFormat` to add these flags, and I don't think we want that. Moving this call to `SetFlags` would fix that.

Reply:
That was the way this working before this pr. `SetFlagsWithFormat` was just setting a default value for Format.

Reply:
Actually though, I can see what you are saying. I'll make the change.